src/syft/lib/petlib/__init__.py

```diff
@@ -0,0 +1,40 @@
+# stdlib
+from typing import Any
+from typing import Dict
+from typing import Union
+
+# third party
+import petlib
+
+# syft relative
+from . import ec  # noqa: 401
+from ...ast.globals import Globals
+from .allowlist import allowlist
+
```

Comment:
You will need to adhere to the loadable library API, check something like `tenseal/__init__.py`:

```python
LIB_NAME = "tenseal"
PACKAGE_SUPPORT = {
    "lib": LIB_NAME,
    "python": {"max_version": (3, 9, 99)},
}


def create_ast(client: TypeAny) -> Globals:
    ast = Globals(client=client)
```